app/styles/mixins.scss

```diff
@@ -25,6 +25,11 @@
   -moz-transition: $string;
 }
 
+@mixin animation($string) {
+  animation: $string;
+  -webkit-animation: $string;
```

Comment:
lolol i am impressed.

these are the only two necessary for `animation`: http://www.w3schools.com/cssref/css3_pr_animation.asp